docs/manual/en/introduction/How-to-run-things-locally.html

```diff
@@ -64,6 +65,24 @@ <h3>Servez</h3>
 				</p>
 			</div>
 
+			<h3>Node.js five-server</h3>
```

Comment:
How about removing this section and just add the `<li>` in the above listing?

Otherwise if more live servers are added in the future, the page will get even more crowded.

Reply:
The listing above says "**Plugins** for popular code editors". But you clearly have to make the difference between `five-server` and `five-server-vscode`. One is the npm package the other is the extension/plugin for VSCode.

_You could remove "Live Server (for VSCode)" since "Five Server (for VSCode)" is a replacement._

> Otherwise if more live servers are added in the future, the page will get even more crowded.

Well, there are **only 2** Node.js servers listed. "Node.js five-server" and "Node.js http-server".

Reply:
Oh okay! I was not aware that you are listing two different projects.